Update Dutch translations with file from Toon. #390

lemonboston opened the pull request:
I can't of course check the translation correctness but the diff looked good. 
Just replaced "..." sequences with ellipsis character as Android suggested.

dmfs commented:
you can use translate.google.com to see if a translation looks correct.
Please approve when you're done.

lemonboston commented:
I've gone over the changes one by one and made some changes:
- removed the dots (...) from all of the as at some places they were clearly incorrect, at other places probably questionable, but it seemed better to remove them
- many changes seemed to be similar or even same meanings according to google, I didn't touch those
- there were a few where it was clearly incorrect, translating the resource name instead of the content, like "Edit task title" in Dutch, I replaced them with the previous version
- made a few other common sense changes

I don't know the origination of this new translation but if it's from a Dutch person and not generated with some tool, I suppose we can assume the changes which result is similar English meaning are improvements. And with that and with this review, I think it is probably a better version then the previous one. (Some English, previously untranslated words have also been translated in here)
